### ISSUE TITLE
Text is hard to read due to low color contrast in “Our Reach in Numbers” section

### ISSUE THREAD

**Issue by AnantaAgarwal** [Nikhilrsingh/car-transport-service#798]
## Bug Description
In the “Our Reach in Numbers” section, some text is not easy to read. The text color is too light for the dark and colorful background, so the words are not clearly visible.

## Steps to Reproduce
1.Open the website.
2. Click on More -> Our Network 
3. Scroll to the “Our Reach in Numbers” section.
4. Look at the text below the heading and numbers.

## Expected Behavior
The text should have good color contrast and be easy to read on the background.

## Screenshots

<img width="1440" height="546" alt="Image" src="https://github.com/user-attachments/assets/e95be782-0aea-4de6-b5d4-6052436edf5e" />
